com.github.sdbg.debug.ui/src/com/github/sdbg/debug/DebugLaunch.java

```diff
@@ -0,0 +1,37 @@
+package com.github.sdbg.debug;
+
+import com.github.sdbg.debug.ui.internal.DartUtil;
+import com.github.sdbg.debug.ui.internal.chrome.ChromeLaunchShortcut;
+import com.google.gdt.eclipse.core.sdbg.IDebugLaunch;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IResource;
+import org.eclipse.jdt.core.IJavaProject;
+import org.eclipse.jdt.core.JavaCore;
+
+public class DebugLaunch implements IDebugLaunch {
```

Comment:
I was thinking more interface assessors could be added like launchChromeMobile(...) instead of adding more extension points, but if you want I'll do this.